src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -467,12 +466,6 @@ uct_rc_mlx5_common_post_send(uct_rc_mlx5_iface_common_t *iface, int qp_type,
 
     ucs_assert(qp_type == iface->super.super.config.qp_type);
 
-#if HAVE_TL_DC
```

Comment:
do we still need to pass qp type to this function?

Reply:
> do we still need to pass qp type to this function?

yes, it is used in lines 467 & 480

Reply:
467 assert could be removed, but 480 is quite ugly and not easy to fix.. ok